docs/rfc/rfc-1.adoc

```diff
@@ -285,6 +285,58 @@ and blinding factor, and then request that TBTC gets minted ASAP. Checking the
 dApp, they can see that they should expect TBTC in their provided wallet
 address in 3 hours with no further interaction.
 
+==== Minting From Unswept Balances
```

Comment:
Here we discuss minting from unswept balances. We should update the 'Minting And Unminting' chapter which suggests that minting is done only from swept balances. We could also consider mentioning how `unsweptBalance` gets updated in case of immediate mint in the `Account Balances` section.